### PR TITLE
Favoriting Progress

### PR DESCRIPTION
**Pull request by aniyahb**

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -60,16 +60,112 @@ app.post('/login', async(req,res) =>{
 })
 
 // Logout Endpoint
-app.post('/logout', (req,res) =>{
-    res.session.destroy(error => {
-        if(error){
-            return res.status(500).json({ error:'Failed to log out' })
+// app.post('/logout', (req,res) =>{
```
Comment (KatsiarynaMashokha) [inline backend/index.js]: pls do not commit any of the unused code

File: frontend/src/Modal/Modal.jsx [hunk 2]
```diff
@@ -5,6 +5,10 @@ import RecipeList from "../RecipeList/RecipeList.jsx";
 
 const Modal = (props) => {
```
Comment (KatsiarynaMashokha) [inline frontend/src/Modal/Modal.jsx:6]: you can just destructure in here instead of the next line

File: frontend/src/Modal/Modal.jsx [hunk 3]
```diff
@@ -5,6 +5,10 @@ import RecipeList from "../RecipeList/RecipeList.jsx";
...
     const { pickedRecipe, closeModal} = props;
```
Comment (KatsiarynaMashokha) [inline frontend/src/Modal/Modal.jsx]: are you using `closeModal` in here? If not pls remove

File: frontend/src/Modal/Modal.jsx (hunk 2 above)
Comment (KatsiarynaMashokha) [inline frontend/src/Modal/Modal.jsx]: you can destructure right here instead of using props down

File: frontend/src/RecipeCard/RecipeCard.jsx [hunk 4]
```diff
@@ -20,19 +59,14 @@ const handleClick = () => {
...
-                            setIsFav(!isFav);
-                            }}>
-                        {isFav ?
-                            <i className="fa-solid fa-heart"></i> :
-                            <i className="fa-regular fa-heart"></i>
-                    }
-                    </a>
+                <a className="favoriteIcon" onClick={handleAddFavorite}>
 
+                    {isFav ?
```
Comment (KatsiarynaMashokha) [inline frontend/src/RecipeCard/RecipeCard.jsx]: only the className is changing here based on `isFav`: you can do a ternary operator only for the className and not an element itself, smth like: `<i className={isFav? "fa-solid fa-heart" : "other-class-name"}></I>`

File: frontend/src/Modal/Modal.jsx (hunk 3 above)
Comment (aniyahb) [reply frontend/src/Modal/Modal.jsx]: I'm using closeModal on line 16.

**Review by KatsiarynaMashokha — APPROVED** [review]

**Pull request merged**